internal/telemetry/metrics.go

```diff
@@ -144,8 +155,11 @@ func (c *collector) Collect(ch chan<- prometheus.Metric) {
 		log.Warn().Err(err).Msg("unable to collect datastore statistics")
 	}
 
+	logicalChecksCount := loadLogicalChecksCount()
```

Comment:
Given that loading zeroes things out, are we concerned at all about the value failing to be sent in this block? Or is a dropped metric send here and there not gonna matter too much for what this metric is supposed to do?

Reply:
A dropped metric is probably fine; this is a rough estimate anyway